### ISSUE TITLE
[DCAN] fMRIPrep / NiBabies version compatibility

### ISSUE THREAD

**Issue by madisoth** [PennLINC/xcp_d#881]
fMRIPrep: >= nipreps/fmriprep:23.0.2 (current release on Dockerhub)

NiBabies: nipreps/nibabies:mcribs for now, then plan on 23.1.x once mcribs is merged into main branch
- Needs to support derivatives made both w/ and w/o `--surface-recon-method mcribs`

(On our end we still need to test all of these, and then share derivatives to test with)
